ex9/js/ajaxGet.js

```diff
@@ -0,0 +1,23 @@
+define(function() {
+    return function(url, callback) {
+      //url = '';
+      $.ajax({
+        type: 'GET',
+        url: url,
+        statusCode: {
+          404: function() {
+            alert( "page not found" );
+          }
+        },
+        error: function(fff){
+            alert( "it is an error" + fff );
+            console.log(fff);
```

Comment:
Once you understand what is the argument, rename fff to it. Also, the doc for error says what this param is. See http://api.jquery.com/jquery.ajax/